app/classifier/tasks/dropdown/index.cjsx

```diff
@@ -115,12 +140,12 @@ module.exports = React.createClass
     @setState optionsKeys: optionsKeys
 
   getOptions: (i) ->
-    select = @props.task.selects[i]
+    select = @state.selects[i]
```

Comment:
I'm finding that this change gives me `option: false` for all options, created or not.